src/xgov-dapp/src/features/vote/VoteResults.tsx

```diff
@@ -63,6 +88,39 @@ export const VoteResults = ({
     return clone
   }, [votingRoundMetadata, snapshot, isLoadingVotingRoundData, isLoadingVotingRoundResults])
 
+  const reserveList = useMemo<Question[] | undefined>(() => {
+    if (votingRoundMetadataClone === undefined) {
+      return undefined
```

Comment:
Could return empty array to keep down on chaining with `?.`

Reply:
you're absolutely right, will do

Reply:
just noticed I didn't use `?.` with `reserveList` and I didn't get any ts warning. how come?

Reply:
`reserveList &&` is a valid check for TS, it's effectively `reserveList?.length`